pkg/exporter/scripts.go

```diff
@@ -56,8 +57,14 @@ func runScript(timeout float64, enforced bool, args []string) (string, int, erro
 	}
 	cmd = exec.CommandContext(ctx, args[0], args[1:]...)
 
-	if timeout > 0 {
+	// Set environments variables
+	if len(env) > 0 {
+		cmd.Env = env
+	} else {
 		cmd.Env = os.Environ()
+	}
```

Comment:
Maybe `os.Environ()` can always be used and then the additional environment variables can be appended, sth. like

```suggestion
	cmd.Env = os.Environ()
	if len(env) > 0 {
		cmd.Env = append(cmd.Env, env...)
	}
```